doc/nrf/releases/release-notes-latest.rst

```diff
@@ -141,6 +141,25 @@ Matter (Project CHIP)
 
   * Renamed occurrences of Project CHIP to Matter.
 
+Bluetooth Mesh
+--------------
+
+* Added:
+
+  * Support for vendor specific mesh model :ref:`bt_mesh_silvair_enocean_srv_readme`.
+  * API function to manually store pending RPL entries in the persistent storage without waiting for the timeout.
+
+* Updated:
+
+  * Samples are using :option:`CONFIG_NVS` instead of :option:`CONFIG_FCB` as default storage backend.
+  * Renamed the following Kconfig options:
+
+    * ``CONFIG_BT_MESH_LIGHT_CRTL_STORE_TIMEOUT`` renamed to :option:`CONFIG_BT_MESH_MODEL_SRV_STORE_TIMEOUT`, and default value is set to 0.
+
+  * The configuration setting for acknowledged message have been moved into Kconfig to make them public.
+  * Added persistent storage to the Scheduler model to restore previously configured entries on power up.
+  * Fixed several bugs.
```

Comment:
Is there a reason why we don't want to be specific about the fixes? If we're talking about just a few bugs, we might as well list them?

Reply:
We should really lock through them then. And check if each one off them may have a impact on the user. Maybe we just should remove this line, and check if there is bug that is important for the user to know about.